kvdb/src/mock_param_dict_impl.cpp

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "kvdb/rocksdb_impl.h"
+#include <thread>
+#include <iterator>
+#include <fstream>
+#include <algorithm>
+#include <sstream>
+
+std::vector<FileReaderPtr> ParamDict::GetDictReaderLst() {
+    return this->file_reader_lst_;
+}
+
+void ParamDict::SetFileReaderLst(std::vector<std::string> lst) {
+    for (size_t i = 0; i < lst.size(); i++) {
+        FileReaderPtr fr = std::make_shared<FileReader>();
+        fr->SetFileName(lst[i]);
+        this->file_reader_lst_.push_back(fr);
+    }
+}
+
+std::vector<float> ParamDict::GetSparseValue(std::string feasign, std::string slot) {
+    auto BytesToFloat = [](uint8_t* byteArray){
+        return *((float*)byteArray);
+    };
+    //TODO: the concatation of feasign and slot is TBD.
+    std::string result = front_db->Get(feasign + slot);
+    std::vector<float> value;
+    if (result == "NOT_FOUND") 
+        return value;
+    uint8_t* raw_values_ptr = reinterpret_cast<uint8_t *>(&result[0]);
+    for (size_t i = 0; i < result.size(); i += 4) {
```

Comment:
Here you are assuming float is of size 4bytes. Better use `i += sizeof(float)`?
Anyways try avoid hard coded magic number

Reply:
covert all related 4 to sizeof(float). thank you.